controller/customdomain_controller.go

```diff
@@ -277,7 +277,9 @@ func (r *CustomDomainReconciler) Reconcile(ctx context.Context, request ctrl.Req
 				return reconcile.Result{}, err
 			}
 			isAWS := *cloudPlatform == "AWS"
-			if isAWS {
+			lbType := customIngress.Spec.EndpointPublishingStrategy.LoadBalancer.ProviderParameters.AWS.Type
```

Comment:
This will always default to nil based on the above definition of `customIngress`. Where are you updating the CRD to allow for a different LB type?

Reply:
Good point. Thanks!